oms-scomacs-solution/azuredeploy.parameters.json

```diff
@@ -5,10 +5,10 @@
     "workspaceName": {
       "value": "GEN-UNIQUE-10"
     },
-    "serviceTier": {
+    "pricingTier": {
       "value": "Free"
```

Comment:
can you set the parameter for pricingTier to be something else than "Free"? This to ensure a valid deployment in the test subscription.